tests/features/user/register.feature

```diff
@@ -60,3 +60,27 @@ Feature: User registration
       | subject   | Your Joinup account was created successfully.                                                                                   |
       | body      | The Joinup Support Team created your account on Joinup. Please log in through the following link in order to set your password. |
     And the account for SuperUser should be active
+
+  Scenario: A user account whose first and last names are the identical is
+    deleted after creation but the user is receiving all notification as if the
+    account would have been created.
+
+    Given I am an anonymous user
+    And I go to "/user/register"
```

Comment:
Let's go through the interface
```
And I am on the homepage
And I click "Create new account"
```

Reply:
I don't see the value this change could bring. However, I'm not against changing it.